Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Github5623.xaml

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
I had to submit this XAML file because I could not wire up bindings in code behind. @hartez Not sure if it was me, but there might be an issue with `SetBinding` call for `ItemsSource`. Would be great to have tests in place for this.